packages/flutter/lib/src/rendering/list_wheel_viewport.dart

```diff
@@ -759,27 +762,27 @@ class RenderListWheelViewport
       _layoutChild(lastChild!, childConstraints, ++currentLastIndex);
     }
 
-    offset.applyViewportDimension(_viewportExtent);
-
     // Applying content dimensions bases on how the childManager builds widgets:
     // if it is available to provide a child just out of target range, then
     // we don't know whether there's a limit yet, and set the dimension to the
     // estimated value. Otherwise, we set the dimension limited to our target
     // range.
+    // Test by 'infinite scrolling child builder with lower and upper limits' at
+    // 'list_wheel_scroll_view_test.dart'.
```

Comment:
Can you put this comment in the test itself? If a future change breaks the test, it will be helpful to have that context there as well.